src/components/structures/VideoRoomView.tsx

```diff
@@ -62,6 +62,10 @@ const VideoRoomView: FC<IProps> = ({ room, resizing }) => {
         }
     }, [room, widgetStoreReady, widgetLoaded]); // eslint-disable-line react-hooks/exhaustive-deps
 
+    // We'll also take this opportunity to fix any stuck devices
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
do we need to disable the linter here? It seems like it'll probably be smart enough to figure out the dependencies.

Reply:
I disabled it since the linter thought that `store.connected` should be a dependency, so have added a comment to that effect